packages/microsoft-reactnative-sampleapps/index.windows.js

```diff
@@ -14,10 +14,14 @@ import {
   UIManager,
   View,
 } from 'react-native';
-import { NativeModules } from 'react-native';
 
-const CustomUserControlCS = requireNativeComponent('CustomUserControlCS');
+import { NativeModules, NativeEventEmitter } from 'react-native';
+
+// Creating event emitters
+const SampleModuleCSEmitter = new NativeEventEmitter(NativeModules.SampleModuleCS);
```

Comment:
>new NativeEventEmitter [](start = 30, length = 22)

Does it create an emitter for the global RCTEventEmitter?

Reply:
All instances of `NativeEventEmitter` share a pointer internally to a singleton RCTDeviceEventEmitter. So even if you (as in your previous sample) create a lazy callable module that returns a NativeEventEmitter, you're still sharing the same namespace for event names.